pkg/ee/resource-quota/default-quota-controller/controller.go

```diff
@@ -230,3 +227,14 @@ func withSettingsEventFilter() predicate.Predicate {
 		},
 	}
 }
+
+func projectQuotaReconcilerFactory(resourceQuota *kubermaticv1.ResourceQuota) reconciling.NamedResourceQuotaReconcilerFactory {
+	return func() (string, reconciling.ResourceQuotaReconciler) {
+		return resourceQuota.Name, func(existing *kubermaticv1.ResourceQuota) (*kubermaticv1.ResourceQuota, error) {
+			existing.Spec = resourceQuota.Spec
+			existing.Labels = resourceQuota.Labels
+			existing.Annotations = resourceQuota.Annotations
```

Comment:
This would remove any other labels the admin or any 3rd party service in the seed clluster might be setting. Maybe it's smarter if we merge our values into whatever is in `existing`?

Reply:
Good point, I've updated the logic.